Sources/CoreData/Repository/ObjectListRepository.swift

```diff
@@ -14,6 +14,9 @@ public protocol ObjectListRepository {
 	var frc: NSFetchedResultsController<ObjectType> { get }
 	func refresh(then handler: @escaping (Result<[ObjectType], Error>) -> Void)
 	func findOldItems() -> [ObjectType]
+
+	@available(iOS 13.0, *)
+	func refresh() async -> Result<[ObjectType], Error>
```

Comment:
There's no need for this, you already provide an async version below.

BUT instead I'd recommend leaving the protocol "as is", and creating a new `AsyncObjectListRepository` protocol:
- sub-protocol of this one
- as requirement add the async `refresh` function
- and has a default implementation for `refresh` (non-async) that calls the async version

For an idea on how to do this, check https://github.com/vapor/fluent-kit/blob/main/Sources/FluentKit/Concurrency/AsyncMigration.swift